specreduce/extract.py

```diff
@@ -294,6 +325,9 @@ class HorneExtract(SpecreduceOperation):
     image: NDData
     trace_object: Trace
     bkgrd_prof: Model = field(default=models.Polynomial1D(2))
```

Comment:
i would rename this `background_profile` for consistency with `spatial_profile`.

Reply:
should we do this in another PR so it has its own change log entry for clarity since this changes the interface?

Reply:
yeah, that's appropriate.